docs/run-a-node/setup/install.md

```diff
@@ -77,6 +77,8 @@ Installing the devtools is simple and no additional entries need to be added for
 
 See the examples below to understand how to install the deb and rpm packages.
 
+> Note that if installing using the updater script (see the section *Installing with Other Linux Distros*) that all the binaries are downloaded together, i.e., there is not a separate devtools archive file or package.
```

Comment:
Even though we didn't change these in the last release, I would clarify with package names, e.g. something like:

Most tools are included in the node binary package and do not require a separate install. There are a few additional tools (such as pingpong) in a separate tools package.

Reply:
Just added this.  Sorry for the delay.